### ISSUE TITLE
Trend moderation: Add 'Allow instance'

### ISSUE THREAD

**Issue by nichu42** [mastodon/mastodon#24585]
### Pitch

Trending post moderation has the options to "Allow post" and "Allow author".
I'd like to have third option, "Allow instance", to auto-allow all posts from a specific instance in the future.

### Motivation

Now that we have media companies joining the Fediverse with their own instances (e.g., zdf.social), this would prevent the work of allowing all authors individually, but to allow all trending posts from trusted media sources.

**Comment by yelvington** [comment]
Notably, "Trending Links" has an "Allow publisher" feature.

**Comment by nichu42** [comment]
Thanks, but I know. This is about something else.

**Comment by ThisIsMissEm** [comment]
This seems reasonable, but I've absolutely no idea how we'd implement it at present, because the feature currently works by setting a flag on the Accounts table. We don't actually have an Instance's table, so we can't set a flag there (we just have a view).

Perhaps once #31247 is done it, might be possible on the resulting table there? Or we create a new table specifically for this.

**Comment by todoneunl** [comment]
Duplicate of #31496.
